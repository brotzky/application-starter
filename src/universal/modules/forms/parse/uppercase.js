export default function(value) {
  return value ? value.toUpperCase() : '';
}
