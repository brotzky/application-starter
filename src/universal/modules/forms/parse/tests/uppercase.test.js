import uppercase from '../uppercase';

describe('[Form Field Parsing] uppercase', () => {
  describe('uppercase()', () => {
    it('should return an uppercase version of a string', () => {
      expect(uppercase('abc')).toMatchSnapshot();
    });
  });
});
