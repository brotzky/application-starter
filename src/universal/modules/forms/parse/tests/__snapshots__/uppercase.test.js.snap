// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Form Field Parsing] uppercase uppercase() should return an uppercase version of a string 1`] = `"ABC"`;
